packages/manager/apps/hycu/src/constants.ts

```diff
@@ -7,3 +7,6 @@ export const packTypeLabel = {
   'hycu-cloud-vm-pack-250': '250 VMs',
   'hycu-cloud-vm-pack-500': '500 VMs',
 } as const;
+
+export const PREFIX_LICENSE_FILE_NAME = 'license-hycu-' as const;
+export const SUFFIX_LICENSE_FILE_NAME = '.dat' as const;
```

Comment:
Not sure it's useful here to specify `as const` like in an "object vs enum" case

Reply:
Why not consider using a kind of name template instead ?
```suggestion
export const LICENSE_FILE_NAME_TEMPLATE = 'license-hycu-{filename}.dat' as const;
```